node-example/taylor.js

```diff
@@ -0,0 +1,99 @@
+const plotly = require('plotly')('username', 'api_key');
```

Comment:
Can the example use plotly offline (without auth) or another graph library that doesn't require accounts?

Reply:
Plotly cannot be used offline. I'll use another library that doesn't require online accounts